app/views/account/tenants/edit.html.slim

```diff
@@ -0,0 +1,9 @@
+h1.col-md-6.col-md-offset-3 Edit Tenant
+.row
+  .col-md-3.col-md-offset-3
+    = simple_form_for @tenant, url: account_tenant_path,
+                                defaults: { input_html: { class: 'form-control' } } do |f|
+      = f.input :name
+      = f.input :website, hint: 'Without "http://" or "https://"'
```

Comment:
this form MUST be extracted to partial. Do not repeat yourself (DRY).
I see same code in NEW view

Reply:
done